cves/CVE-2013-0861.yml

```diff
@@ -169,10 +185,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: "Michael Niedermayer mitigated the memory corruption"
+  automated: false
```

Comment:
I think the `answer` field should explain how the vulnerability was found, not how it was fixed. When a commit for a fix has this line:
"Found-by: Mateusz "j00ru" Jurczyk and Gynvael Coldwind",
it indicates that the vulnerability was found by a fuzzer. See https://gynvael.coldwind.pl/?id=524

Reply:
Fixed to say such that Michael Niedermayer found the vulnerability.